src/components/toolbar/DropdownFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import {useState} from 'react';
+import {
+  Dropdown,
+  DropdownGroup,
+  DropdownToggle,
+  DropdownItem,
+  ToolbarItem,
+} from '@patternfly/react-core';
+
+export function DropdownFilter() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onFocus = () => {
+    const element = document.getElementById('toolbar-dropdown-filter');
+    element.focus();
+  };
+
+  const onSelect = () => {
+    setIsOpen(false);
+    onFocus();
+  };
+
+  const dropdownItems = [
+    <DropdownItem key="col-name">Name</DropdownItem>,
+    <DropdownItem key="col-name">Visibility</DropdownItem>,
+  ];
+
+  return (
+    <ToolbarItem spacer={{default: 'spacerNone'}}>
+      <Dropdown
+        onSelect={onSelect}
+        toggle={
+          <DropdownToggle
+            id="toolbar-dropdown-filter"
+            onToggle={() => setIsOpen(isOpen)}
+            icon={<i className="fa fa-filter"></i>}
```

Comment:
use the Icon component directly from patternfly instead of `<i>` see the React component for the icon in https://www.patternfly.org/v4/guidelines/icons/  looks like it is `< FilterIcon />`